dolphinscheduler-task-plugin/dolphinscheduler-task-mlflow/src/main/java/org/apache/dolphinscheduler/plugin/task/mlflow/MlflowTask.java

```diff
@@ -197,6 +206,33 @@ private Map<String, Property> getParamsMap() {
 
     }
 
+    public int checkDockerHealth() throws Exception {
+        logger.info("checking container healthy ... ");
+        int exitCode = -1;
+        String[] command = {"sh", "-c", String.format(MlflowConstants.DOCKER_HEALTH_CHECK, mlflowParameters.getContainerName())};
+        for(int x = 0; x < MlflowConstants.DOCKER_HEALTH_CHECK_TIMEOUT; x = x+1) {
+            String status;
+            try {
+                status = OSUtils.exeShell(command).replace("\n", "").replace("\"", "");
+            } catch (Exception e) {
+                status = String.format("error --- %s", e.getMessage());
+            }
+            logger.info("container healthy status: {}", status);
+
+            if (status.equals("healthy")) {
+                exitCode = 0;
+                logger.info("container is healthy");
+                return exitCode;
+            }else {
+                logger.info("The health check has been running for {} seconds", x * 5);
+                Thread.sleep(5000);
```

Comment:
```suggestion
                ThreadUtils.sleep(Constants.SLEEP_TIME_MILLIS * 5);
```

Reply:
Thanks for your suggestion, I will fix that

Reply:
Already fix